tests/system/action/motion_state/test_sort.py

```diff
@@ -0,0 +1,44 @@
+from openslides_backend.permissions.permissions import Permissions
```

Comment:
Add 2 more tests:
- an id that is present in the meeting is missing in the payload
- the payload has an extra id which does not exist in the meeting

Reply:
Added two additional test: one with extra id and one with missing id.